internal/handlers/telegram/handler.go

```diff
@@ -56,7 +57,8 @@ func stickerHandler(tg *Client, u tgbotapi.Update) {
 	formatted := tg.Settings.Prefix + u.Message.From.UserName +
 		tg.Settings.Suffix + u.Message.Sticker.Emoji
 
-	tg.sendToIrc(formatted)
+	// Trim unexpected trailing whitespace
```

Comment:
@kennedy this whitespace is actually purposely done. All messages start with a space after the ending suffix. Message should say
```
<test> Random Text
```
and _**not**_:
```
"<test>Random Text"
```

Reply:
According to the formatted string, it was "<test>Random Text ". The trim removes the trailing " " at the end

Reply:
fixed